### DISCUSSION TITLE
Alternative sound channel option

### DISCUSSION THREAD

**Discussion by Doni49** [Nailik/rhasspy_mobile#334] [Ideas]
I have Tasker installed.  I would like an option in the settings to play sounds via a channel other than media.  This way when Rhasspy is going to play sounds, I can just have Tasker turn the media volume all the way down while Rhasspy plays on the Alarm channel.

This would be much easier than getting Tasker to pause and then resume playing the music.

**Comment by Nailik** [comment]
Currently it plays sound as CONTENT_TYPE_MUSIC (when selecting sound) or CONTENT_TYPE_SONIFICATION (when selecting notification).

Maybe this issue could be resolved with the other Discussion https://github.com/Nailik/rhasspy_mobile/discussions/333 when rhasspy simply stops media by it self (aka requests audio focus)
